package-lock.json

```diff
@@ -34,27 +36,29 @@
       "integrity": "sha512-kmv8CGrPfN9SwMwrkiBK9VTQYxdFQEGe0BmQk+M8io56P9KNzpAxcWE/1fxJj7uouwN4kXF0BHW8DNlgx+wtCg=="
     },
     "@azure/core-auth": {
-      "version": "1.0.0",
-      "resolved": "https://registry.npmjs.org/@azure/core-auth/-/core-auth-1.0.0.tgz",
-      "integrity": "sha512-DcXqxt0O0HDhYTcrN16e/hJutFkFTM8fQq6bJbHuDh5AUbDUKFrF64hj5RXATukThOszGFrR1MWJOGZPSo7fiA==",
+      "version": "1.0.2",
+      "resolved": "https://registry.npmjs.org/@azure/core-auth/-/core-auth-1.0.2.tgz",
+      "integrity": "sha512-zhPJObdrhz2ymIqGL1x8i3meEuaLz0UPjH9mOq9RGOlJB2Pb6K6xPtkHbRsfElgoO9USR4hH2XU5pLa4/JHHIw==",
       "requires": {
         "@azure/abort-controller": "^1.0.0",
-        "@azure/core-tracing": "1.0.0-preview.5",
+        "@azure/core-tracing": "1.0.0-preview.7",
+        "@opentelemetry/types": "^0.2.0",
         "tslib": "^1.9.3"
       }
     },
     "@azure/core-http": {
```

Comment:
>core-http [](start = 12, length = 9)

So the azure/app-configuration reference doesn't pull in this package?

Reply:
Whoops this is package lock. 

---
In reply to: [356710441](https://github.com/Azure/AppConfiguration-Sync/pull/4#discussion_r356710441) [](ancestors = 356710441)